msal-dotnet-articles/advanced/managed-identity.md

```diff
@@ -1,51 +1,79 @@
 ---
 title: Managed identity with MSAL.NET
+description: "How to use Azure managed identities in MSAL.NET applications."
 ---
 
 # Managed identity with MSAL.NET
 
 >[!NOTE]
->This is an experimental feature for 1st party use only.
+>This feature is experimental and available from [MSAL.NET](https://www.nuget.org/packages/Microsoft.Identity.Client/) version 4.51.0.
 
-## Managed Identity for Confidential Client
+A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. [Managed identities](/azure/active-directory/managed-identities-azure-resources/overview) in Azure eliminate the need for developers to manage these credentials manually. MSAL.NET supports acquiring tokens through the managed identity capability when used with applications running inside Azure infrastructure, such as:
 
-A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. Managed identities eliminate the need for developers to manage these credentials.
+* [Azure App Service](https://azure.microsoft.com/products/app-service/) (API version `2019-08-01` and above)
+* [Azure VMs](https://azure.microsoft.com/free/virtual-machines/)
+* [Azure Arc](/azure/azure-arc/overview)
+* [Azure Cloud Shell](/azure/cloud-shell/overview)
+* [Azure Service Fabric](/azure/service-fabric/service-fabric-overview)
 
-## How to use managed identity in MSAL
+For a complete list, refer to [Azure services that can use managed identities to access other services](/azure/active-directory/managed-identities-azure-resources/managed-identities-status).
+
+## How to use managed identities
+
+There are two types of managed identities available to developers - **system-assigned** and **user-assigned**. You can learn more about the differences in the [Managed identity types](/azure/active-directory/managed-identities-azure-resources/overview#managed-identity-types) article. MSAL.NET supports acquiring tokens through both. Standard [MSAL.NET logging](/azure/active-directory/develop/msal-logging-dotnet) is possible to keep track of requests and related metadata.
+
+Prior to using managed identities from MSAL.NET, developers must enable them for the resources they want to use through Azure CLI or the Azure Portal.
+
+## Examples
+
+For both user-assigned and system-assigned identities, developers can use the <xref:Microsoft.Identity.Client.ManagedIdentityApplicationBuilder> class. Because the feature is experimental, using <xref:Microsoft.Identity.Client.BaseAbstractApplicationBuilder%601.WithExperimentalFeatures(System.Boolean)> is required.
```

Comment:
Seems like there a few tricks to use, like these xref, when working with the docs. Unfortunately these don't render in GitHub PRs properly...

Reply:
Yeah, that's a trade-off with proper cross-doc references :/